.travis.yml

```diff
@@ -16,9 +16,9 @@ script:
   xargo/build.sh
 - |
   # Test plain miri
-  cargo build --locked --release --features "cargo_miri" &&
-  cargo test --release --all &&
-  cargo install --features "cargo_miri"
+  cargo build --locked --release --all-features &&
+  cargo test --locked --release --all-features --all &&
+  cargo install --locked --release --all-features
```

Comment:
install is `--release` by default, I guess it doesn't really matter though.

Reply:
Yeah I thought so, but then it felt odd to have that flag everywhere except here.

Reply:
... or maybe not. Oh well.^^